pkg/plugins/models.go

```diff
@@ -288,3 +288,13 @@ type Permission struct {
 	Action string `json:"action"`
 	Scope  string `json:"scope"`
 }
+
+type Extensions struct {
+	Links []*ExtensionsLink `json:"links"`
+}
+
+type ExtensionsLink struct {
+	Id          string `json:"id"`
```

Comment:
```suggestion
	ID          string `json:"id"`
```

Reply:
Only a question: is this a "general" Go convention (to use `ID` instead of `Id`) or more a convention in our codebase?

Reply:
Yeah it's a pretty common convention I would say. In Grafana we switched this a couple years back, so it would be much more consistent with the styling in Grafana at the very least!